core/core.go

```diff
@@ -59,16 +59,17 @@ func (c *Core) InsertChain(blocks types.Blocks) (int, error) {
 	nodeCtx := common.NodeLocation.Context()
 	domWait := false
 	for i, block := range blocks {
-		isCoincident := c.sl.engine.IsDomCoincident(block.Header())
 
-		// Write the block body to the db.
-		rawdb.WritePendingBlockBody(c.sl.sliceDb, block.Header().Root(), block.Body())
+		isCoincident := c.sl.engine.IsDomCoincident(block.Header())
```

Comment:
Can you just revert this section of the diff? I think you meant to undo the changes, but whitespace and stuff is still showing a diff

Reply:
check now